### PR TITLE
Remove crossbeam

### PR DESCRIPTION
**Pull request by juchiast**
Previous pull request is closed due to conflict.

This version should be called v0.1.1, instead of v0.2 in the branch name.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 [package]
 name = "async_mutex"
-version = "0.1.0"
-authors = ["Do Duy <juchiast@gmail.com>"]
+version = "0.1.1"
+authors = ["real <real@freedomlayer.org>"]
```
Comment (realcr) [inline Cargo.toml]: @juchiast : You should add yourself and Kamyuen Tse (kevin.xjy@gmail.com) as authors. After all you two made this work.

**Comment by realcr** [comment]
You are fast!

**Review by realcr — CHANGES_REQUESTED** [review]
You can add yourself and kamx to the authros list (:

**Comment by juchiast** [comment]
I added me and kamx in 9eaa562.
And solve https://github.com/realcr/async_mutex/issues/5 in 90871c4

**Comment by realcr** [comment]
Thanks!

**Pull request merged**